project.mml

```diff
@@ -538,9 +536,11 @@ Layer:
         (SELECT
             way,
             landuse,
+            military,
             way_area/NULLIF(!pixel_width!::real*!pixel_height!::real,0) AS way_pixels
           FROM planet_osm_polygon
           WHERE landuse = 'military'
+            OR military = 'danger_area'
             AND building IS NULL
```

Comment:
In SQL, AND has precedence over OR, so this will be parsed as `landuse = 'military' OR (military = 'danger_area' AND building IS NULL)`. The intended meaning is of course `(landuse = 'military' OR military = 'danger_area') AND building IS NULL` so we are missing a pair of brackets here.